### PR TITLE
Add `chainerx.sigmoid` docs

### PR DESCRIPTION
**Pull request by crcrpar**

### CODE REVIEW

**Comment by hvy** [comment]
Jenkins, test this please.

File: chainerx/_docs/routines.py [hunk 1]
```diff
@@ -1149,6 +1149,24 @@ def _docs_math():
...
+    _docs.set_doc(
+        chainerx.sigmoid,
+        """sigmoid(x)
+Element-wise sigmoid logistic function.
+
+Args:
+    x (~chainerx.ndarray): Input array.
+
+Returns:
+    :class:`~chainerx.ndarray`: Returned array: :math:`f(x) = (1 + \\exp(-x))^{-1}`
```
Comment (hvy) [inline chainerx/_docs/routines.py]: Could you fix the style error? https://travis-ci.org/chainer/chainer/jobs/519521073

**Comment by hvy** [comment]
Thanks, for this PR. There's a typo in the title if you could fix it :)

**Comment by crcrpar** [comment]
lol, thanks.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by crcrpar** [comment]
Hmm, does Macro OP PR has something to do with these failures?

**Comment by hvy** [comment]
That seems to be the case. Let me rerun the CI.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
CI failure is unrelated, LGTM!

**Pull request merged**